### ISSUE TITLE
Can not transpile for ios 9

### ISSUE THREAD

**Issue by yringler** [emberjs/ember.js#18589]
Scoping keywords (let, const) do not work on ios9, and can not be removed from vendor.js

*Replication Repo*
https://github.com/yringler/transpile-repro

I tried to use targets.js to specify support for ios9, and explicilty included the @babel/plugin-transform-block-scoping plugin in ember-cli-build.js, but vendor.js still has instances of let, causing the app to crash on ios9.

`let` is removed from vendor.js, except for this section (partially copied below)

```
;(function() {
/*!
 * @overview  Ember - JavaScript Application Framework
 * @copyright Copyright 2011-2019 Tilde Inc. and contributors
 *            Portions Copyright 2006-2011 Strobe Inc.
 *            Portions Copyright 2008-2011 Apple Inc. All rights reserved.
 * @license   Licensed under MIT license
 *            See https://raw.github.com/emberjs/ember.js/master/LICENSE
 * @version   3.14.2
 */

/*globals process */
let define, require, Ember;

// Used in @ember/-internals/environment/lib/global.js
mainContext = this; // eslint-disable-line no-undef

(function() {
  let registry;
  let seen;
```

**Comment by yringler** [comment]
Possibly introduced by [this commit](https://github.com/emberjs/ember.js/commit/4a5b372e0f1b3c70412b4afd101d6ae8707d9987)
https://github.com/emberjs/ember.js/blob/master/packages/loader/lib/index.js

**Comment by pzuraq** [comment]
This bug has been fixed on the 3.12 LTS, pending release, and 3.13+. We will not be backporting this fix to 3.8 LTS since Safari 9 is outside of our browser support policy and 3.8 is only receiving critical security fixes at the point, per our normal LTS support policy.

**Comment by yringler** [comment]
Thank you.
Closing as duplicate of https://github.com/ember-cli/ember-cli/issues/8938
